dbt_subprojects/dex/seeds/trades/shadow_sonic_base_trades_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_month,block_date,block_time,block_number,token_sold_amount_raw,token_bought_amount_raw,token_sold_address,token_bought_address,tx_hash,evt_index
+sonic,shadow,1,2025-01-01 00:00,2025-01-16 00:00,2025-01-16 00:25,4074192,5000000000000000000,342625313575091371,0x039e2fb66102314ce7b64ce5ce3e5183bc94ad38,0x3333b97138d4b086720b5ae8a7844b1345a33333,0xe1f2af42b3146ba103604d38c04b69c5f7e9dcdc6cf30fe55779a9e3a7e1eec5,3
```

Comment:
If we remove version `1`, then we need to remove this line from the seed as well.